go.mod

```diff
@@ -362,3 +362,5 @@ require (
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
 	sigs.k8s.io/structured-merge-diff/v4 v4.2.3 // indirect
 )
+
+replace github.com/Microsoft/go-winio => github.com/tailscale/go-winio v0.0.0-20230929170319-cac332cedf3a
```

Comment:
we can't use replace in oss go.mod. only in corp.

Reply:
Well, what's the extent of the fork changes? If it changes the API shape at all, we definitely can't.

If it only fixes some bugs with the existing API we _could_, but then people running e.g. tsnet servers on Windows wouldn't get the fixes. (people using this module from their own go module)

So sadly I think we need to rename in our fork.

Reply:
Yeah it's an API change. Sadly, the upstream PR was actually accepted, but whoever is in charge of merging it has not done so.

Reply:
OK, I've renamed our fork.